lib/arjdbc/jdbc/adapter.rb

```diff
@@ -528,24 +428,10 @@ def self.prepared_statements?(config)
             false # off by default - NOTE: on AR 4.x it's on by default !?
       end
 
-      if @@suble_binds = Java::JavaLang::System.getProperty('arjdbc.adapter.suble_binds')
-        @@suble_binds = Java::JavaLang::Boolean.parseBoolean(@@suble_binds)
-      else
-        @@suble_binds = ActiveRecord::VERSION::MAJOR < 4 # due compatibility
-      end
-      def self.suble_binds?; @@suble_binds; end
-      def self.suble_binds=(flag); @@suble_binds = flag; end
-
       private
 
-      # @private Supporting "string-subling" on AR 4.0 would require {#to_sql}
-      # to consume binds parameters otherwise it happens twice e.g. for a record
-      # insert it is called during {#insert} as well as on {#exec_insert} ...
-      # but that than leads to other issues with libraries that save the binds
-      # array and run a query again since it's the very same instance on 4.0 !
-      def suble_binds(sql, binds)
-        sql
-      end
+      # @deprecated no longer used
+      def suble_binds(sql, binds); sql end
```

Comment:
We do not need to deprecate if this is dead or dying.  50.x is major release.

Reply:
heritage of cherry-picking the backports from support - will remove, thanks